root/programs/C#/Frameworks/Infrastructure/CustomControl/PropertyGrid/CheckType.cs

```diff
@@ -28,30 +28,44 @@
 //*  日時        更新者            内容
 //*  ----------  ----------------  -------------------------------------------------
 //*  2016/01/28  Sai               Corrected IsIndispensabile property spelling
+//*  2017/01/31  西野 大介         "Indispensable" ---> "Required"
 //**********************************************************************************
 
-// System
 using System;
-using System.Collections.Generic;
 using System.ComponentModel;
-using System.Drawing;
-using System.Text;
-
-// System.Web
-using System.Web;
-using System.Web.UI;
-using System.Web.UI.WebControls;
 
 namespace Touryo.Infrastructure.CustomControl
 {
     /// <summary>チェック タイプ</summary>
     [TypeConverter(typeof(CheckTypeConverter))]
     public class CheckType
     {
+        /// <summary>ビット・フィールド</summary>
+        [Flags]
+        public enum BF
```

Comment:
@yusukemaegawa 

Since this is an inner enum of CheckType class, the user will not deal directly with it.
Therefore, this name is in an abstract state. However, I will make corrections if necessary.